1-python-wrapper/README.md

```diff
@@ -0,0 +1,15 @@
+# 1 - Python wrapper
+
+We take the iris data set and train a simple classification model for it.
+
+* Training code should go in `training`
+* A module with a `predict` function should be put into `model`
+* A flask interface and main app should be put in `wrapper`
+
+## Expected format
+
+predict: { "f1": 0, "f2" :1, "f3": 2 } => { "class": "predicted_class" }
+
+## How to start
+
+* Document how to start the flask api here
```

Comment:
Could you actually document how to start the flask API here? Just the set of bash commands required to install the venv, requirements, and start the application. As well as an example curl request to make test the API.

Reply:
Yes!